src/pages/billing/NewBillingProjectWizard/StepWizard/Step.ts

```diff
@@ -0,0 +1,37 @@
+import { CSSProperties } from 'react'
+import { section } from 'react-hyperscript-helpers'
+import colors from 'src/libs/colors'
+import { StepHeader } from 'src/pages/billing/NewBillingProjectWizard/StepWizard/StepHeader'
+
+
+function stepBanner(active: boolean): CSSProperties {
+  return {
+    borderRadius: '8px',
+    boxSizing: 'border-box',
+    padding: '1.5rem 2rem',
+    marginTop: '1rem',
+    display: 'flex',
+    flexDirection: 'column',
+    width: '100%',
+    border: active ? `1px solid ${colors.accent()}` : `1px solid ${colors.accent(0.2)}`,
+    backgroundColor: active ? colors.light(0.5) : colors.light(0.3),
+    boxShadow: active ? '0 0 5px 0 rgba(77,114,170,0.5)' : 'none'
+  }
+}
+
+export interface StepProps {
+  isActive: boolean
+  title?: string
+  introText?: React.ReactNode[]
```

Comment:
Why is `introText` a ReactNode (instead of a string)?

Reply:
So you could do things like text formatting, adding links, etc.

`ReactNode` is just anything react can render, which includes strings.

I switched to just composing it more explicitly using the `StepHeader` as the first child, which is more in-line with standard react patterns (I just checked, and all of the current usages do this).
I'll go ahead and remove title and introText as props, since they aren't really used anymore.